website/docs/r/org.html.markdown

```diff
@@ -3,16 +3,21 @@ layout: "vcd"
 page_title: "vCloudDirector: vcd_org"
 sidebar_current: "docs-vcd-resource-org"
 description: |-
-  Provides a vCloud Director Organization resource. This can be used to create and delete an organization.
+  Provides a vCloud Director Organization resource. This can be used to create  delete, and update an organization.
 ---
 
 # vcd\_org
 
-Provides a vCloud Director Org resource. This can be used to create and delete an organization.
+Provides a vCloud Director Org resource. This can be used to create, update, and delete an organization.
 Requires system administrator privileges.
 
 Supported in provider *v2.0+*
 
+!> **Warning:** Up to version 2.4, there were two bugs in the handling of this resource. If you have existing resources
+created in versions 2.0 to 2.4, you should re-create them using `terraform state rm` and `terraform import`
```

Comment:
1.
"If you have existing resources created in versions 2.0 to 2.4," ->
"If you have existing vcd_org resource created with earlier versions,"

2.
"you should re-create them using `terraform state rm` and `terraform import`
(See *Upgrading Org resources to 2.5* below for the steps)."
->
"you should re-create them by following *Upgrading Org resources to 2.5* section below"

Reply:
done